proparse/src/main/java/org/prorefactor/core/NodeTypes.java

```diff
@@ -94,6 +96,10 @@ private NodeTypes() {
     return kwSet;
   }
 
+  /*static void add(int type, String fullText, int flags) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3ACommentedOutCodeLine)